third_party/move/tools/move-package/src/lib.rs

```diff
@@ -109,6 +109,10 @@ pub struct BuildConfig {
     #[clap(name = "test-mode", long = "test", global = true)]
     pub test_mode: bool,
 
+    /// Whether to override the standard library with the given version.
+    #[clap(long = "override-std", global = true, value_parser)]
+    pub override_std: Option<String>,
```

Comment:
Yeah as @wrwg mentioned, you could change this to `pub override_std: Option<StdVersion>` if you implement the required traits for `StdVersion`.

Reply:
done